data/account.go

```diff
@@ -58,6 +58,20 @@ func GetAccount(uuid string) (*Account, error) {
 	return account, err
 }
 
+// GetAccountByLogin returns an account with matching login.
+// Returns an error if no account with such login exists.
+func GetAccountByLogin(login string) (*Account, error) {
+	const q = `SELECT * FROM Accounts a WHERE a.login=$1`
+
+	account := &Account{}
+	err := database.Get(account, q, login)
+	if err != nil && err != sql.ErrNoRows {
+		panic(err)
```

Comment:
I think I really like the idea of not bubbling up errors that normally should really happen at all. But then the only actual "error" ever returned is `sql.ErrNoRows` and there is not really much information in that. Shouldn't we just use the "comma, ok" version then `(*Account, bool)`? Or just one return value (with `nil` indicates not found)?